samples/bot-commands-menu/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-commands-menu-nodejs",
+    "source": "officeDev",
+    "title": "Create a commands menu",
+    "shortDescription": "This sample add a command menu with a dropdown list of commands for your bot.Users can select a command from the list, which will insert the command string into the message box. Then, they can select Send to execute the command.",
```

Comment:
Same comments as above for the text

Reply:
Updated below Content -

This sample add a command menu with a dropdown list of commands for your bot. Users can select a command from the list, which will insert the command title into the compose box. Then, they can select Send to execute the command.